src/styles/blocks/product.scss

```
@@ -0,0 +1,68 @@
+.product {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    align-items: center;
+    padding-left: 3px;
+    margin: 0 auto 28px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    margin: 0;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: var(--light-text-color);
+    margin-bottom: 7px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-text {
+      color: var(--light-text-color);
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
+
+  &__buy {
+    background-color: var(--blue-accent);
+    border: none;
+    color: var(--white-color);
+    padding: 11px 70px;
```

```suggestion
    padding: 11px;
```
Consider centering the text differently. The text for the button may change